cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -214,4 +219,48 @@ protected void addService(Service service) {
   protected void addService(String name, HttpServiceHandler handler, HttpServiceHandler...handlers) {
     configurer.addService(new BasicService(name, handler, handlers));
   }
+
+  /**
+   * Schedules the specified {@link Workflow}
+   * @param schedule the schedule to be added for the Workflow
+   * @param workflowName the name of the Workflow
+   */
+  protected void scheduleWorkflow(Schedule schedule, String workflowName) {
+    scheduleWorkflow(schedule, workflowName, new HashMap<String, String>());
+  }
+
+  /**
+   * Schedule the specified {@link Workflow}
+   * @param scheduleName the name of the Schedule
+   * @param cronTab the crontab entry for the Schedule
+   * @param workflowName the name of the Workflow
+   */
+  protected void scheduleWorkflow(String scheduleName, String cronTab, String workflowName) {
+    String scheduleDescription = scheduleName + " with crontab " + cronTab;
```

Comment:
Here you should call scheduleWorkflow(scheduleName, cronTab, workflowName, Collections.emptyMap())